src/power.jl

```diff
@@ -200,15 +200,10 @@ coefficient of `a`.
         return nothing
     end
 
-    # Relevant for positive integer r, to avoid round-off errors
-    if isinteger(r) && (k-l0 > r*findlast(a) > 0)
-        @inbounds c[k-l0] = zero(c[0])
-        return nothing
-    end
-
```

Comment:
I can't remember if these lines are important or not?

Reply:
Good catch! Sorry yeah, they do look important for integer powers; I'll put them back

Reply:
Thanks!